docs/Hooks.md

```diff
@@ -4,61 +4,26 @@
 
 Hooks are registered with the `fastify.addHook` method and allow you to listen to specific events in the application or request/response lifecycle. You have to register a hook before the event is triggered otherwise the event is lost.
 
-## Request/Response Hooks
-
 By using the hooks you can interact directly inside the lifecycle of Fastify. There are seven different Hooks that you can use *(in order of execution)*:
-- `'onRequest'`
-- `'preParsing'`
-- `'preValidation'`
-- `'preHandler'`
-- `'preSerialization'`
-- `'onError'`
-- `'onSend'`
-- `'onResponse'`
-
-Example:
-```js
-fastify.addHook('onRequest', (request, reply, done) => {
-  // some code
-  done()
-})
-
-fastify.addHook('preParsing', (request, reply, done) => {
-  // some code
-  done()
-})
-
-fastify.addHook('preValidation', (request, reply, done) => {
-  // some code
-  done()
-})
-
-fastify.addHook('preHandler', (request, reply, done) => {
-  // some code
-  done()
-})
-
-fastify.addHook('preSerialization', (request, reply, payload, done) => {
-  // some code
-  done()
-})
+- [onRequest](#onRequest)
+- [preParsing](#preParsing)
+- [preValidation](#preValidation)
+- [preHandler](#preHandler)
+- [preSerialization](#preSerialization)
+- [onError](#onError)
+- [onSend](#onSend)
+- [onResponse](#onResponse)
```

Comment:
I would improve this list also with the Application Hooks:

- Request/Reply Hooks
  - onRequest
  - etc...
- Application Hooks
  - onRegister
  - etc...

so the user will have a clear overview of all the hooks, that is not right now as you pointed out 😂

Reply:
Sure! I'll do make it today maybe.

Reply:
@Eomm Changed! Can check the new version, please?